### DISCUSSION TITLE
Prints now printing left of center since update

### DISCUSSION THREAD

**Discussion by EhRabz** [mriscoc/Ender3V2S1#1108] [Q&A]
ever since I installed this firmware, my prints seem to be shifted to the left, the bedsize is set proper in the physical settings, what do I do?

**Comment by mriscoc** [comment]
1. Set your physical dimensions ([Min/Max X/Y/Z Positions](https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides#finding-the-minmax-position)) before changing the Bed size (Advanced / Physical settings). Check if you need to correct the parking position (Advanced / Park Head).
2. Set your probe offsets (Advanced / Probe settings).
3. Set your mesh inset (Advanced / Mesh leveling / Mesh inset) according to your physical dimensions and probe offsets, you can automatically do it pressing Maximize area and then Center area. Always set the mesh inset only after setting physical limits and bed size.
4. If you have a Runout sensor, configure the active mode (Advanced / Filament Setting /Runout Active), if not, leave the "Enabled Runout" disabled.
5. Store your settings and Control/Reboot printer.
6. **Check if your slicer's printer profile has the same physical settings.**
